### ISSUE TITLE
missing tab-titles @ tabs from content

### ISSUE THREAD

**Issue by xerc** [laxap/bootstrap_grids#25]
any idea why the tabs are not rendered @ http://microsite.alpha-realestate.de/#ankaufsprozess

```
typo3/cms                     7.6.15
typo3-ter/gridelements        7.0.5
laxap/bootstrap-grids         1.3.1 GIT
maxserv/replacecontent        2.0.4 GIT
typo3-ter/sourceopt           0.9.0
typo3-ter/realurl             2.1.5
typo3-ter/iconfont            0.9.0
in2code/powermail             3.11.2
```

**Comment by laxap** [comment]
I doesn't look like tabs but it works like tabs. :) What is exactly the problem on this page? Looks good to me.

![screenshot](https://cloud.githubusercontent.com/assets/10398927/25348748/8a9dd710-291f-11e7-8f83-df0df15ec707.jpg)

**Comment by xerc** [comment]
merged with the issue #24
